compositor_pipeline/src/pipeline/output/mp4.rs

```diff
@@ -269,9 +279,22 @@ fn create_packet(
         }
     }?;
 
+    // Starting output PTS from 0
+    let timestamp_offset = match start_pts {
+        Some(start_pts) => *start_pts,
+        None => {
+            *start_pts = Some(chunk.pts);
+            chunk.pts
+        }
+    };
```

Comment:
```suggestion
    let timestamp_offset = offset.get_or_insert_with(||Some(chunk.pts));
```

or add a method to Stream struct that takes current chunk pts and returns the offset